sdk/core/azure-core/src/http/curl/curl_connection_private.hpp

```diff
@@ -162,7 +179,7 @@ namespace Azure { namespace Core { namespace Http {
        * @brief Destructor.
        * @details Cleans up CURL (invokes `curl_easy_cleanup()`).
        */
-      ~CurlConnection() override { curl_easy_cleanup(this->m_handle); }
+      ~CurlConnection() override {}
```

Comment:
are you sure about this ? seems that outside the curldeleter this will leak.
Or are you trying to help me choose the next stress test ?

Reply:
`CurlConnection::m_handle` is a `unique_CURL`. And a `unique_CURL` is a `std::unique_ptr`. 

If `std::unique_ptr` is leaking, we have a WAY bigger issue than this.

More importantly, the code crashes if I keep the call to `curl_easy_cleanup` :). Ask me how I know :).